rpc/src/rpc.rs

```diff
@@ -2285,13 +2285,14 @@ fn get_encoded_account(
     pubkey: &Pubkey,
     encoding: UiAccountEncoding,
     data_slice: Option<UiDataSliceConfig>,
+    overwrite_accounts: Option<&HashMap<Pubkey, AccountSharedData>>,
```

Comment:
Please add a code comment above this field to explain that it's only used for simulation results.

Reply:
okay! [50cba61](https://github.com/solana-labs/solana/pull/34619/commits/50cba61aa40eb7601d02327f5638648b684f4805)

Reply:
I see a comment on `get_parsed_token_account()`. Could you please duplicate it here?

Reply:
```suggestion
    // only used for simulation results
    overwrite_accounts: Option<&HashMap<Pubkey, AccountSharedData>>,
```